core/liquidity/v2/sla.go

```diff
@@ -87,8 +88,7 @@ func (e *Engine) calculateCurrentTimeBookFraction(now, start time.Time, s time.D
 	if lNano > 0 {
 		timeBookFraction = num.DecimalFromInt64(s.Nanoseconds()).Div(num.DecimalFromInt64(lNano))
 	}
-
-	return timeBookFraction
+	return num.MinD(num.DecimalOne(), timeBookFraction)
```

Comment:
Can we also cap from below by `0.0`?

Reply:
negative values here necessitate time-travel, which is currently unsupported by vega. :)